test/onnx/test_pytorch_onnx_shape_inference.py

```diff
@@ -283,5 +285,112 @@ def test_reduce_prod_without_axes(self):
         self.run_test(g, reduce_prod.node(), expect_tensor("Long", shape=(1,)))
 
 
+class TestONNXCustomOpShapeInference(common_utils.TestCase):
+
+    opset_version = _constants.ONNX_MAX_OPSET
+
+    def test_single_customOp_setType(self):
```

Comment:
I would name the test with expected behavior: something along the lines of

`test_setType_maintains_output_shape_for_single_custom_op`

to make tests clear

Reply:
Done